.eslintrc

```diff
@@ -32,9 +33,12 @@
     "space-before-function-paren": [2, "never"],
     "space-before-blocks": [2],
     "no-multi-spaces": [2],
+    "no-whitespace-before-property": [2],
+    "no-unexpected-multiline": [2],
     "space-infix-ops": [0, {"int32Hint": false}],
     "quotes": [2, "single"],
     "dot-notation": [2, {"allowKeywords": false}],
+    "operator-linebreak": [2, "after"],
```

Comment:
This [rule](http://eslint.org/docs/rules/operator-linebreak.html) is already seen across the repo. It is now enforced.